spec/api-process-spec.js

```diff
@@ -0,0 +1,26 @@
+const assert = require('assert')
+
+describe('process module', function () {
+  describe('process.getCPUUsage()', function () {
+    it('returns a cpu usage object', function () {
+      const cpuUsage = process.getCPUUsage()
+      assert.equal(typeof cpuUsage.percentCPUUsage, 'number')
+      assert.equal(typeof cpuUsage.idleWakeupsPerSecond, 'number')
+    })
+  })
+
+  describe('process.getIOCounters()', function () {
+    it('returns an io counters object', function () {
+      if (process.platform !== 'win32') {
```

Comment:
Should this be `process.platform == 'darwin'` since this spec should work on Linux as well?